api/graph.go

```diff
@@ -63,3 +66,23 @@ func (g *graph) E() interfaces.Edge {
 func (g *graph) String() string {
 	return g.name
 }
+
+// multiParamQuery creates a query based on the given (optional) parameters.
+// The query is the name of the query method that supports 0..* parameters.
+// Examples:
+//    q1:=multiParamQuery(".out","label1","label2") ==> generates ".out('label1','label2')"
+//    q2:=multiParamQuery(".out") ==> generates ".out()"
+func multiParamQuery(query string, params ...string) interfaces.QueryBuilder {
+	if len(params) == 0 {
+		return NewSimpleQB(fmt.Sprintf("%s()", query))
+	}
+
+	qStr := ""
+	for _, p := range params {
+		qStr += fmt.Sprintf("\"%s\",", p)
```

Comment:
After doing some research, I've discovered that the `fmt.Sprintf()` has one of the slower performances. So rather than iterating through and using `fmt.Sprintf()` to concatenate, we can use `strings.Join()` where
it will just be `qStr := strings.Join(params, "\",\"")`
then we can simply to `qStr := fmt.Sprintf("%s(\"%s\")",query,qStr)`
this will avoid calling the `fmt.Sprintf()` multiple times

Reply:
We could possibly use the `strings.Builder` for a faster performance not only runtime but also reduce memory complexity as well

Reply:
Good hint. I adjusted the PR accordingly.